Server/LauncherClient/Launcher.cs

```diff
@@ -122,8 +125,16 @@ private void game_start_timer_Tick(object sender, EventArgs e)
         public void SetConfigValue(string key, string value)
         {
             Configuration configuration = ConfigurationManager.OpenExeConfiguration(ConfigurationUserLevel.None);
+
+            if(key == "Secret")
+            {
+                //string encryptedValue = encryption.Encrypt(value);
+                string encryptedValue = MachineKeyEncryption.Protect(value, "Secret");
```

Comment:
The "Secret" part here should be something more unique. Possibly something like `$"Secret for computer {computerKey}"`

Reply:
That makes sense, I'll adjust that

Reply:
Fixed in commit 92bb6e50243d6c75d4b505f3d08b5a86f6231997